app/Components/Sudoku Board/Components/Cell.tsx

```diff
@@ -8,7 +8,7 @@ let fallbackHeight = 30;
 interface RenderCellProps {
   entry: any; // todo find some way to derive this from type instad of duplicate
   type: CellType;
-  onClick: (r: number, c: number) => void;
+  onClick: (r: number, c: number, event: any) => void;
```

Comment:
not clear to me what event is here, could you add a comment above this interface briefly explaining some of it?  Also, this file is light on comments in general, might be a good time to add comments above getNoteContents and getCellContents functions explaining them (and the format cell contents are stored in)?

Reply:
I had the docs in toggleSelectCell function, and I don't like the idea of copy-pasting it in multiple places.
I added comments for getNoteContents and getCellContents
![image](https://github.com/Sudokuru/Frontend/assets/60023923/85454f7d-c263-477f-889b-ce96b6fb119f)